spec/restaurant_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'restaurant'
+
+describe Restaurant do
+  subject(:restaurant) { described_class.new(menu: menu) }
+  let(:menu) { double :menu, view: list }
+  let(:list) { [
+    {"Chips" => 3},
+    {"Salad" => 4},
+    {"Chicken" => 7},
+    {"Fish" => 9},
+    {"Chef's special" => 15}
+  ] }
+#we have a real class, restaurant, with an injected fake class, menu. When our fake menu gets the command 'view' it just returns the list
```

Comment:
Metrics/LineLength: Line is too long. [137/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)